src/sentry/relay/config/__init__.py

```diff
@@ -125,7 +125,7 @@ def get_filter_settings(project: Project) -> Mapping[str, Any]:
         # 423 - There was an error while hydrating. Because the error happened outside of a Suspense boundary, the entire root will switch to client rendering.
         # 425 - Text content does not match server-rendered HTML.
         error_messages += [
-            "https://reactjs.org/docs/error-decoder.html?invariant={418,419,422,423,425}"
```

Comment:
Is this a regex ?
If it is a regex I don't think it is correct 
if you want to select "https://..... =" followed by one of the numbers than you would need something like:
```
https://reactjs.org/docs/error-decoder.html\?invariant=(418|419|422|423|425){1}
```

which in python you can write as:
```
# normal string
error_message = "https://reactjs.org/docs/error-decoder.html\\?invariant=(418|419|422|423|425){1}"
# raw string (no double backslash)
error_message = r"https://reactjs.org/docs/error-decoder.html\?invariant=(418|419|422|423|425){1}"

Reply:
I tested it with Joris in Rust and it works

Reply:
It's a glob, not a regex.